.github/linters/.yaml-lint.yml

```diff
@@ -4,16 +4,21 @@ extends: default
 rules:
   braces:
     max-spaces-inside: 1
+    level: error
```

Comment:
Isn't it possible to define some sort of "default" level for every rule?

Reply:
uhh nice point, will check the docs.

Reply:
hmm, I didn't see a configuration section for it. https://yamllint.readthedocs.io/en/stable/configuration.html#configuration